nix/overlay.nix

```diff
@@ -49,7 +49,7 @@ let
           "${ghc-typelits-natnormalise}"
           { };
       # doctest-parallel = hprev.callHackage "doctest-parallel" "0.3.1" { };
-      doctest-parallel = hprev.callHackageDirect { pkg = "doctest-parallel"; ver = "0.3.1"; sha256 = "sha256-j0mk4RTEau6Cl90E8vMWZiA/ldRRurtLSYbGGE+q/vk="; } {};
+      # doctest-parallel = hprev.callHackageDirect { pkg = "doctest-parallel"; ver = "0.3.1"; sha256 = "sha256-j0mk4RTEau6Cl90E8vMWZiA/ldRRurtLSYbGGE+q/vk="; } {};
```

Comment:
What's up with this? Can't we simply remove it?

Reply:
Yeah, I guess we can just remove it indeed!